slicops/ui_api/screen.py

```diff
@@ -95,18 +99,23 @@ async def api_screen_start_button(self, api_args):
         ux = self._session_ui_ctx()
         self._set_acquire(1)
         self._button_setup(ux, True)
+        ux.plot.auto_refresh = True
         return await self._return_with_image(ux)
 
     async def api_screen_stop_button(self, api_args):
         ux = self._session_ui_ctx()
+        ux.plot.auto_refresh = False
         self._set_acquire(0)
         self._button_setup(ux, False)
         return self._return(ux)
 
     async def api_screen_ui_ctx(self, api_args):
         ux = self._session_ui_ctx()
         # TODO(robnagler) if accepting ui_ctx, then need to update valid_values here
-        return self._return(ux)
+        return self._return(ux).pkupdate(
+            # TODO(pjm): need a better way to load a resource for a sliclet
+            layout=pkyaml.load_file(pkresource.file_path("static/layout/screen.yaml")),
```

Comment:
I don't think we need "static/" in the name. For now, this should be named "ui_api/screen.yml".

Reply:
I moved it back a level. I did move the field def into schema/screen.yaml, so now package_data contains directories "layout" and "schema"